### ISSUE TITLE
Progress pop-up minimise 

### ISSUE THREAD

**Issue by knapp-amics** [Bruker-Amics/bugs_features#32]

**Comment by Matt-Power** [comment]
From Teams:
The progress pop-up window in Process is always on top so gets in the way of all other work when active. If you click on it to move it, Process jumps to the fore and you have to relocate the application you were working on. Add ability to minimise (and also ensure that Process does not grab focus when the pop-up is clicked or moved around).

Window is always on top so really gets in the way when working on multiple instances of Process or any other work which is being done while waiting.

**Comment by van-amics** [comment]
Conflict request: we got a request of making it "always on top" a while back, and the always-on-top progress bar is the result of it.

**Comment by Matt-Power** [comment]
Can we have an option in the preferences then please? When working on several instances of Process, it is really challenging. I lose my place and pick up on a similar but different / wrong sample almost daily.

**Comment by MarkGuo7** [comment]
@Matt-Power You got it:)
Do you need a switch or not-on-top is good enough?

**Comment by Matt-Power** [comment]
Is it possible to just have it on top of the instance it is running from? (I.e. it's doesn't get in the way of other apps or instances of amics.) If that is not possible, not on top is good enough for me as long as it doesn't get hidden (there was a build of amics where it would be impossible to get to certain message boxes because they were on underneath others). I realise that others may prefer something different though so an option in the preferences is always useful.

**Comment by MarkGuo7** [comment]
@Matt-Power Got it, and will do as you suggested. Thanks

**Comment by MarkGuo7** [comment]
There it is :)

![image.png](https://images.zenhubusercontent.com/60a59e61f0254ad3e154481a/ec7d7347-5284-49b8-b89d-75c47c9c144f)

**Comment by Matt-Power** [comment]
Yay, thank you!

**Comment by Matt-Power** [comment]
I've just switched this off (great!) but unfortunately, now when one instance is running a process, I cannot get to a different instance. When I click on the correct instance on taskbar, it defaults to the instance processing. I've tried moving the window (both the progress window and the actual window) to see if I can click on the one underlying but it's not visible. I have also tried ctrl tab and windows tab but when I select the instance that I want, it defaults back to the one processing. I'll test it once the process has finished to see if it is the same. This is on latest beta 2325.

OK, quick update, this only seems to be the case if both instances are doing something. Seems to be intermittent - the one doing the processing occasionally grabs focus (even when you are doing something in the other instance).
